CMakeLists.txt

```
@@ -1,9 +1,21 @@
 cmake_minimum_required(VERSION 3.8)
 project(qualisys_cpp_sdk)
 
+# By default set release configuration
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel." FORCE)
+endif()
+
```

Instead of adding and forcing the compilation of the shared library you can simply add
```suggestion
option(SUFFIX_SO_VERSION "Suffix library name with its version" ON)
```
See suggestion below for the deletion of the `SHARED` parameter.